lib/tracer.js

```diff
@@ -73,6 +73,7 @@ module.exports = class Tracer {
         err.toString()
           .replace(/[\r\n]+/g, '<br/>')
           .replace(/"/g, '\\"')
+          .replace(/\\/g, '\\\\')
```

Comment:
L75 has replaced to `\"`, then you replace `\` to `\\`, does it has some problem?